mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Postgrex.Mixfile do
   def project do
     [app: :postgrex,
      version: "0.9.2-dev",
-     elixir: "~> 1.0",
+     elixir: "~> 1.1",
```

Comment:
I'm not sure we want to push postgrex to 1.1 yet, it just came out. Can you reimplement `GenServer.whereis`?